src/PointwiseFunctions/AnalyticData/SelfForce/GeneralRelativity/CircularOrbitCoeffs.cpp

```diff
@@ -0,0 +1,646 @@
+Areal[6][2]=0;
+Areal[6][3]=0;
+Areal[6][4]=0;
+Areal[6][5]=0;
+Areal[6][6]=(-4*a*a)/pow(a*a+r*r,2)+(4*a*a*a*a)/(r*pow(a*a+r*r,2))+(6*a*a*r)/pow(a*a+r*r,2)-(8*r*r)/pow(a*a+r*r,2)+(2*r*r*r)/pow(a*a+r*r,2)+(16*r*r)/pow(a*a+2*r*r+a*a*cos(2*th),2)-(4*a*a)/((a*a+r*r)*(a*a+2*r*r+a*a*cos(2*th)))-(4*a*a*r)/((a*a+r*r)*(a*a+2*r*r+a*a*cos(2*th)))+(4*r*r)/((a*a+r*r)*(a*a+2*r*r+a*a*cos(2*th)))-(4*r*r*r)/((a*a+r*r)*(a*a+2*r*r+a*a*cos(2*th)));
+Areal[6][7]=0;
+Areal[6][8]=(2*a*a*sin(2*th))/((a*a+r*r)*(r*r+a*a*pow(cos(th),2)))-(a*a*a*a*sin(2*th))/(r*(a*a+r*r)*(r*r+a*a*pow(cos(th),2)))-(a*a*r*sin(2*th))/((a*a+r*r)*(r*r+a*a*pow(cos(th),2)));
+Areal[6][9]=0;
+Areal[7][0]=0;
+Areal[7][1]=0;
+Areal[7][2]=0;
+Areal[7][3]=0;
+Areal[7][4]=0;
+Areal[7][5]=(4*a*a*r*sin(2*th))/((a*a+r*r)*(a*a+2*r*r+a*a*cos(2*th)));
+Areal[7][6]=0;
+Areal[7][7]=(-4*a*a*a*a)/(pow(a*a+r*r,2)*(a*a+2*r*r+a*a*cos(2*th)))+(2*a*a*a*a*a*a)/(r*pow(a*a+r*r,2)*(a*a+2*r*r+a*a*cos(2*th)))+(2*a*a*a*a*r)/(pow(a*a+r*r,2)*(a*a+2*r*r+a*a*cos(2*th)))-(4*a*a*a*a*cos(2*th))/(pow(a*a+r*r,2)*(a*a+2*r*r+a*a*cos(2*th)))+(2*a*a*a*a*a*a*cos(2*th))/(r*pow(a*a+r*r,2)*(a*a+2*r*r+a*a*cos(2*th)))+(6*a*a*a*a*r*cos(2*th))/(pow(a*a+r*r,2)*(a*a+2*r*r+a*a*cos(2*th)))-(8*a*a*r*r*cos(2*th))/(pow(a*a+r*r,2)*(a*a+2*r*r+a*a*cos(2*th)))+(4*a*a*r*r*r*cos(2*th))/(pow(a*a+r*r,2)*(a*a+2*r*r+a*a*cos(2*th)));
+Areal[7][8]=0;
+Areal[7][9]=0;
+Areal[8][0]=0;
+Areal[8][1]=0;
+Areal[8][2]=-((a*a*a*a*a*sin(th))/(r*(a*a+r*r)*pow(r*r+a*a*pow(cos(th),2),2)))+(3*a*a*a*r*sin(th))/((a*a+r*r)*pow(r*r+a*a*pow(cos(th),2),2))+(6*a*r*r*r*sin(th))/((a*a+r*r)*pow(r*r+a*a*pow(cos(th),2),2))-(a*a*a*a*a*cos(2*th)*sin(th))/(r*(a*a+r*r)*pow(r*r+a*a*pow(cos(th),2),2))+(a*a*a*r*cos(2*th)*sin(th))/((a*a+r*r)*pow(r*r+a*a*pow(cos(th),2),2));
+Areal[8][3]=0;
+Areal[8][4]=0;
+Areal[8][5]=0;
+Areal[8][6]=(a*a*r*sin(2*th))/((a*a+r*r)*(r*r+a*a*pow(cos(th),2)));
+Areal[8][7]=0;
+Areal[8][8]=(-2*a*a)/pow(a*a+r*r,2)+(2*a*a*a*a)/(r*pow(a*a+r*r,2))+(4*a*a*r)/pow(a*a+r*r,2)-(6*r*r)/pow(a*a+r*r,2)+(2*r*r*r)/pow(a*a+r*r,2)+(16*r*r)/pow(a*a+2*r*r+a*a*cos(2*th),2)-(4*a*a)/((a*a+r*r)*(a*a+2*r*r+a*a*cos(2*th)))-(4*a*a*r)/((a*a+r*r)*(a*a+2*r*r+a*a*cos(2*th)))+(4*r*r)/((a*a+r*r)*(a*a+2*r*r+a*a*cos(2*th)))-(4*r*r*r)/((a*a+r*r)*(a*a+2*r*r+a*a*cos(2*th)));
+Areal[8][9]=0;
+Areal[9][0]=0;
+Areal[9][1]=0;
+Areal[9][2]=0;
+Areal[9][3]=(4*a*a*a*r*sin(th))/((a*a+r*r)*pow(r*r+a*a*pow(cos(th),2),2))+(12*a*r*r*r*sin(th))/((a*a+r*r)*pow(r*r+a*a*pow(cos(th),2),2))-(4*a*a*a*a*a*pow(cos(th),2)*sin(th))/(r*(a*a+r*r)*pow(r*r+a*a*pow(cos(th),2),2))+(4*a*a*a*r*pow(cos(th),2)*sin(th))/((a*a+r*r)*pow(r*r+a*a*pow(cos(th),2),2));
+Areal[9][4]=0;
+Areal[9][5]=0;
+Areal[9][6]=0;
+Areal[9][7]=0;
+Areal[9][8]=0;
+Areal[9][9]=(2*a*a*a*a)/(r*pow(a*a+r*r,2))+(2*a*a*r)/pow(a*a+r*r,2)-(4*r*r)/pow(a*a+r*r,2)+(32*r*r)/pow(a*a+2*r*r+a*a*cos(2*th),2)-8/(a*a+2*r*r+a*a*cos(2*th));
+return 0;
+}
+
+int getAimag(int m, double a, double /*om*/, double r, double th,
+             double Aimag[10][10]) {
+/* om = m*Omega */
```

Comment:
I would suggest switching to zero-init using std::fill/for loop and then removing all the zero-init. Yes it's generated code, but this is very tedious to look at and you could reduce the number of lines in half